example/README.md

```diff
@@ -123,7 +123,7 @@ You can deploy Calico on Kubernetes cluster with the following command.
 $ kubectl apply -k .
 ```
 
-See [Calico Documentation](https://docs.projectcalico.org/getting-started/kubernetes/self-managed-onprem/onpremises) for details.
+See [Calico Documentation](https://docs.tigera.io/calico/3.25/getting-started/kubernetes/self-managed-onprem/onpremises) for details.
```

Comment:
How about linking to `https://docs.tigera.io/calico/latest/getting-started/kubernetes/self-managed-onprem/onpremises`?
Updating the version here is a bit of a burden.

Reply:
Surely it would be burden to raise the version in the document each time, so I have corrected the url.
b8a45f0ec250d86ffe6ca39dae71406aaca8ad57